### PR TITLE
apache: use fakeroot instead of postinst script

### PR DESCRIPTION
**Pull request by micmac1**
fakeroot support was recently added to OpenWrt. Let's make use of it.

Signed-off-by: Sebastian Kemper <sebastian_ml@gmx.net>

Maintainer: @heil 
Compile tested: master ath79
Run tested: ath79 master, installed package to device, checked that permissions are as expected

Description:
This drops postinst script in favor of using fakeroot.

Kind regards,
Seb

### CODE REVIEW

**Comment by aparcar** [comment]
Thanks for this patch. I somewhat dislike the raw usage of user numbers instead of names. This would however require a `passwd` like file containing a mapping, generated from the USERID in all available feeds. Would you be interested to build such feature?

**Comment by micmac1** [comment]
Hi Paul, my time these days is rather limited. If I had a brilliant quick idea on how to do it I'd do it, though. But currently no such thing comes to mind ;-) Maybe some day in the future.

**Comment by karlp** [comment]
those fixed uids looks absolutely horrific

**Comment by micmac1** [comment]
It's just a number. And it's the same that is used elsewhere for the user creation. As such in my opinion there is nothing horrific about it. It is correct after all and works as expected.

**Comment by aparcar** [comment]
@karlp thanks for your feedback. I'm already working on a easier readable version [here](https://github.com/openwrt/openwrt/pull/3415)

**Comment by aparcar** [comment]
You can now use `apache` instead of a numeric value. Please update this PR.

**Comment by micmac1** [comment]
> You can now use `apache` instead of a numeric value. Please update this PR.

Thanks! Updated.

**Comment by aparcar** [comment]
Great thanks, I'm not the maintainer here so won't merge it. But I approve the change :)

**Review by aparcar — APPROVED** [review]

**Pull request merged**
